os/drivers/input/Kconfig

```diff
@@ -0,0 +1,27 @@
+menu "Touch Driver Support"
```

Comment:
This folder is the `input`, not `touch`. I think input is bigger category than touch. I think this line should be `input driver`